File: SISMUEBLUM C#/ComponenteDeKit.cs
using System;
public class ComponenteDeKit{
	private Producto _producto;
	private double _cantidad;
	
	public ComponenteDeKit(Producto producto, double cantidad){
		this._producto = producto;
		this._cantidad = cantidad;
	}
	
	public Producto producto{get{return this._producto;} set{this._producto = value;}}
	public double cantidad{get{return this._cantidad;} set{this._cantidad = value;}}
	
	~ComponenteDeKit(){}
	
}

File: SISMUEBLUM JAVA/ComponenteDeKit.java
public class ComponenteDeKit{
	private Producto producto;
	private double cantidad;
	
	public ComponenteDeKit(Producto producto, double cantidad){
		this.producto = producto;
		this.cantidad = cantidad;
	}
	
	public Producto getProducto () {return this.producto;}
	public void setProducto (Producto valor) {this.producto = valor;}
	public double getCantidad () {return this.cantidad;}
	public void setCantidad (double valor) {this.cantidad = valor;}
}